app/controllers/application_controller.rb

```diff
@@ -1,12 +1,25 @@
 class ApplicationController < ActionController::Base
+  include Pundit
+  after_action :verify_authorized, unless: :devise_controller?
+
```

Comment:
That's good! We also want to `verify_policy_scoped` for indexes only

Reply:
resolved

Reply:
@JJohnson606 for future reference, let the approver resolve conversation :)